### DISCUSSION TITLE
docs/istio/

### DISCUSSION THREAD

**Discussion by giscus** [ricsanfre/pi-cluster#511] [Announcements]
# docs/istio/

How to deploy service-mesh architecture based on Istio. Adding observability, traffic management and security to our Kubernetes cluster.

https://picluster.ricsanfre.com/docs/istio/

**Comment by SoufianeEddiry** [comment]
First of all thank you for this amazing documentation regarding Istio. I have one question ? did you still use nginx ingress controller ? if not can you share with us how configuring the istio with nginxingress controller
